tools/dataproc-edge-node/README.md

```diff
@@ -0,0 +1,124 @@
+# Dataproc Edge Node
+
+This repository provides scripts for creating Dataproc Edge Nodes in two steps.
+Step 1 creates a Dataproc cluster, installs a configuration script, and captures a VM Image.
+Step 2 launches a GCE VM Instance from the image created by Step 1.
+
+
+## What is an Edge Node?
+
+An Edge Node is a user-facing VM Instance configured to interact with a remote Dataproc cluster. 
+
+The relationship is similar to a reverse proxy placed in front of a web application.
+
+A Dataproc Edge Node can be used as an environment to store data, scripts, code and notebooks.
+
+Because an edge node is not an active member of the cluster and does not execute tasks or contain sensitive data, users can be granted SSH access and local privileges to the edge node with less risk than direct access the cluster.
```

Comment:
You can still ssh from the edge node into the cluster ? If that's the case can you please put a warning here.